pkg/k8s/watchers/endpointsynchronizer.go

```diff
@@ -95,14 +103,6 @@ func (epSync *EndpointSynchronizer) RunK8sCiliumEndpointSync(e *endpoint.Endpoin
 					return fmt.Errorf("Kubernetes apiserver is not available")
```

Comment:
Can this condition (k8s connectivity) change as well, or can we also hoist it out of the controller?

Reply:
Looks like PR has been merged?

Reply:
Hah! Nevermind :-)